plugins/@grouparoo/mailjet/src/lib/client.ts

```diff
@@ -0,0 +1,178 @@
+import * as mailjet from "node-mailjet";
+import Axios from "axios";
+
+class MailjetClient {
+  client: any;
+  apiKey: any;
+  apiSecret: any;
+
+  constructor(apiKey: string, apiSecret: string) {
+    this.apiKey = apiKey;
+    this.apiSecret = apiSecret;
+    this.client = mailjet.connect(apiKey, apiSecret);
+  }
+
+  async createList(listName: string): Promise<any> {
```

Comment:
It looks like there are type defintions for node-mailjet!  If you install them, you probably don't need to type these methods and the types of `equest?.body?.Data` can be determined?

https://www.npmjs.com/package/@types/node-mailjet